openbb_platform/core/openbb_core/provider/standard_models/lbma_fixing.py

```diff
@@ -36,12 +33,6 @@ class LbmaFixingQueryParams(QueryParams):
         description=QUERY_DESCRIPTIONS.get("end_date", ""),
     )
 
-    @field_validator("asset", mode="before", check_fields=False)
-    @classmethod
-    def to_lower(cls, v: Optional[str]) -> Optional[str]:
-        """Convert field to lowercase."""
-        return v.lower() if v else v
-
```

Comment:
Why removing this?

Reply:
There are defined choices for this, and it's better to avoid computed fields in the standard models unless they are really necessary.